doc/code/interfaces/tfe.rst

```diff
@@ -1,3 +1,5 @@
+.. _tfe:
```

Comment:
I was getting lots of build errors from sphinx for these two pages. They had a placeholder in docs, but it was empty, and no TOC entry showed up

Reply:
btw the above change didn't solve the issue :laughing: